### DISCUSSION TITLE
Devex interview with MS Dev Experience expert

### DISCUSSION THREAD

**Discussion by pacificosund** [community/community#74906] [Programming Help]
During one of the interludes, there was an interview with an expert from Microsoft describing qualitative and quantitative developer experience. Please help me find that video where Anjuan Simmons interviewed Nicole Forsgren. Thanks!

**Comment by samus-aran** [comment]
Hey @pacificosund 👋 

Was that today's or yesterday viewing?

If it was today's then all recorded sessions should be available 2 hours after showing. However if it was in between the sessions we may have to dig that out 😅

**Reply by pacificosund** [reply]
It was 8 Nov in the morning sessions somewhere between livestreams. Anjuan Simmons interviewed Nicole Forsgren.

**Reply by pacificosund** [reply]
Hej Samus!

It would be a shame for the interview to be lost in /dev/null. There was some valuable insights that I haven't found in some of Ms Forsgren's other publicly shared information. Please help!
